src/Summoner/Template.hs

```diff
@@ -326,31 +335,28 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
             "http://stackage.org/package/" <> repo <> "/badge/nightly"
         stackLinkNightly :: Text =
             "http://stackage.org/nightly/package/" <> repo
-
-        stackBadges :: Text = memptyIfFalse stack
+        stackBadges :: Text =
             [text|
             [![Stackage Lts](${stackShieldLts})](${stackLinkLts})
             [![Stackage Nightly](${stackShieldNightly})](${stackLinkNightly})
             |]
 
         travisShield :: Text =
-          "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
+            "https://secure.travis-ci.org/" <> owner <> "/" <> repo <> ".svg"
         travisLink :: Text =
-          "https://travis-ci.org/" <> owner <> "/" <> repo
-        travisBadge :: Text = memptyIfFalse travis
+            "https://travis-ci.org/" <> owner <> "/" <> repo
+        travisBadge :: Text = 
```

Comment:
I see that we have a lot of badges. And since we're not going to use `neat-interpolation` for `README.md` file anymore, I think it makes sense to introduce function like `makeBadge` so we can just use 

```haskell
mkBadge "Stackage Lts" stackShieldLts stackLinkLts
```

And same for other badges. Though, you still can use `neat-interpolation` to implement `mkBadge` function itself.